toonz/sources/toonz/shortcutpopup.cpp

```diff
@@ -49,8 +49,10 @@ class ShortcutItem final : public QTreeWidgetItem {
     text.remove("&");
     setText(0, text);
     QString shortcut = m_action->shortcut().toString();
-    if (shortcut != "")
-      setText(1, shortcut);
+	if (shortcut != "")
+		setText(1, shortcut);
+	else
+		setText(1, "");
```

Comment:
Unless I'm missing something really subtle here, you can simply replace the entire `if … else` thing with just `setText(1, shortcut);` since in your `else` branch, `shortcut` would logically always be equal to "" anyway, i.e. you don't have to discriminate between the two cases.